core/arch/arm/plat-synquacer/main.c

```diff
@@ -38,15 +39,37 @@ static struct pl011_data console_data;
 
 register_phys_mem(MEM_AREA_IO_NSEC, CONSOLE_UART_BASE, CORE_MMU_DEVICE_SIZE);
 register_phys_mem(MEM_AREA_IO_SEC, GIC_BASE, CORE_MMU_DEVICE_SIZE);
+register_phys_mem(MEM_AREA_IO_SEC, THERMAL_SENSOR_BASE, CORE_MMU_DEVICE_SIZE);
 
 const struct thread_handlers *generic_boot_get_handlers(void)
 {
 	return &handlers;
 }
 
+static inline uint64_t read_icc_hppir1_el1(void)
```

Comment:
This should be implemented in `core/arch/arm/include/arm64.h` using some of the helper macros, same goes for all other functions like this in this PR.

Reply:
Ok will implement them in core/arch/arm/include/arm64.h.